### PR TITLE
Add counter to runtime viewer

### PR DESCRIPTION
**Pull request by Kyep**
## What Does This PR Do
Alters the format of the "organized" view of the admin runtime error viewer.
Now, a (X) is displayed before each runtime, with X = how many of that runtime have happened this round.
If the count for a single runtime exceeds the max before an error is silenced (9) it displays as "10+".

## Why It's Good For The Game
Makes it easier for staff to find out which runtimes are common and which are rare.

## Images of changes
ytX4lytnsK

## Changelog
:cl: Kyep
add: Runtime viewer (for admins) now shows a count of how many times each runtime error has happened.
/:cl:

### CODE REVIEW

File: code/modules/error_handler/error_viewer.dm [hunk 1]
```diff
@@ -86,7 +86,7 @@ GLOBAL_DATUM(error_cache, /datum/ErrorViewer/ErrorCache)
 		var/datum/ErrorViewer/ErrorSource/error_source
 		for(var/erroruid in error_sources)
 			error_source = error_sources[erroruid]
-			html += "<p class='runtime_list'>[error_source.makeLink(null, src)]<br></p>"
+			html += "<p class='runtime_list'>([error_source.errors.len]) [error_source.makeLink(null, src)]<br></p>"
```
Comment (SteelSlayer) [inline code/modules/error_handler/error_viewer.dm:89]:
```suggestion
			html += "<p class='runtime_list'>([length(error_source.errors)]) [error_source.makeLink(null, src)]<br></p>"
```

**Review by farie82 — APPROVED** [review]

**Comment by AffectedArc07** [comment]
Does this counter include skipped runtimes? If not it's going to be useful up to the number 10, then skipped runtimes will take over.

**Comment by Kyep** [comment]
Updated to use "10+" if errors exceed ERROR_LIMIT.

**Review by Dumbdumn5 — APPROVED** [review]
When I read the title I thought you were somehow adding a counter to the runtime viewer in terms of game balance and I was incredibly confused, but then I realized I was a moron and this is a decent change. :+1:

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**